packages/circuit-ui/components/Avatar/Avatar.tsx

```diff
@@ -103,11 +127,44 @@ export const Avatar = ({
   alt = '',
   variant = 'object',
   size = 'yotta',
+  initials,
   ...props
 }: AvatarProps): JSX.Element => {
+  if (
+    process.env.NODE_ENV !== 'production' &&
+    process.env.NODE_ENV !== 'test' &&
+    variant === 'object' &&
+    initials
+  ) {
+    throw new CircuitError(
+      'Avatar',
+      'The `initials` prop can only be used with the identity `variant`. Remove the `initials` prop or change the `variant` to identity.',
+    );
+  }
+
+  if (
+    process.env.NODE_ENV !== 'production' &&
+    process.env.NODE_ENV !== 'test' &&
+    initials &&
+    initials.length > 2
+  ) {
+    throw new CircuitError(
+      'Avatar',
+      `The \`initials\` prop is ${initials.length} characters long. Shorten it to 1-2 characters.`,
+    );
+  }
+
+  if (variant === 'identity' && !src && initials) {
+    return (
+      <Initials aria-label={alt} role="img" size={size} variant={variant}>
```

Comment:
The initials should have the same semantic role as the image variant.

Reply:
Still announced as "image" with default VO settings 👇 

<img width="824" alt="Default VoiceOver settings: 'Navigate images: always'" src="https://user-images.githubusercontent.com/35560568/194521522-0db0f627-6b87-4830-9d0b-8608c8bb851f.png">

This is why I've added `aria-hidden` to some other components when the alt was empty.

On the other hand maybe that isn't best practice—if the setting is set to _always_ navigate images even with an empty alt, maybe that's a cue that users will want to hear "image" when there is one, no matter if it's basically irrelevant (easier to form a mental model of the UI? Or maybe easier to follow the cursor for SR users with vision impairments but not blind?)

Since all WCAG recommendations mention the empty alt but never adding `aria-hidden`, let's go with this, and we can make other components consistent with the behavior in the future

Reply:
This matches the behavior of an actual image without `alt` text, doesn't it?

If the VO setting is set to "always" announce images we should respect that.

Reply:
> If the VO setting is set to "always" announce images we should respect that.

Yup agreed. The only gotcha here is that this is also the default. I'm not sure it's actually a very good default (if an image is decorative it shouldn't be announced). But that's probably a VO thing and let's follow WCAG recs instead of trying to optimize for a single SR+browser combination